changelog.d/cache-remap-compilation.enhancement.md

```diff
@@ -0,0 +1,3 @@
+Cache result of remap compilation which is called multiple times during vector initialization improving startup time with large remap transforms.
```

Comment:
```suggestion
Vector's start-up time was greatly improved when loading configurations including many `remap` transforms.
```

These changelog entries are intended to be user, rather than developer, facing.